### PR TITLE
fix some bugs about spelling and unless reference

### PR DESCRIPTION
**Pull request by Thor-wl**
Signed-off-by: Thor-wl <13164644535@163.com>

### CODE REVIEW

**Review by wpeng102 — COMMENTED** [review]
/lgtm

File: content/en/docs/tf_on_volcano.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ toc = true  # Show table of contents? true/false
 type = "docs"  # Do not modify.
 
 # Add menu entry to sidebar.
-linktitle = "tensorflow"
+linktitle = "Tensorflow"
```
Comment (jasonliu747) [inline content/en/docs/tf_on_volcano.md:12]: TensorFlow

**Review by jasonliu747 — COMMENTED** [review]
Nice work!

File: content/en/docs/tf_on_volcano.md (hunk 1 above)
Comment (Thor-wl) [reply content/en/docs/tf_on_volcano.md:12]: > TensorFlow

Yes, Let me fix it.

Comment (Thor-wl) [reply content/en/docs/tf_on_volcano.md]: fixed

**Review by william-wang — APPROVED** [review]
/lgtm

**Review by jasonliu747 — APPROVED** [review]
/lgtm

**Review by wpeng102 — APPROVED** [review]
/lgtm

**Review by wpeng102 — APPROVED** [review]

**Pull request merged**